x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -670,17 +672,22 @@ export const selectNewTermsRuleType = () => {
 export const waitForAlertsToPopulate = async (alertCountThreshold = 1) => {
   cy.waitUntil(
     () => {
+      cy.log('Waiting for alerts to appear');
       refreshPage();
-      return cy
-        .get(SERVER_SIDE_EVENT_COUNT)
-        .invoke('text')
-        .then((countText) => {
-          const alertCount = parseInt(countText, 10) || 0;
-          return alertCount >= alertCountThreshold;
-        });
+      return cy.root().then(($el) => {
+        const emptyTableState = $el.find('[data-test-subj="alertsStateTableEmptyState"]');
```

Comment:
NIT: Extract the element to the screens folder.

Reply:
Done. Changes are here: https://github.com/elastic/kibana/pull/149128/commits/ad000e974eacfce9b3f02d9a69af2c5d3057a6ee